libraries/AP_Vehicle/AP_Vehicle.cpp

```diff
@@ -972,6 +972,21 @@ void AP_Vehicle::get_osd_roll_pitch_rad(float &roll, float &pitch) const
 #endif
 }
 
+// Check for pilot input
+bool AP_Vehicle::have_pilot_input() const
```

Comment:
Rather than a vehicle function why not a RC lib function that each vehicle can override in its ` RC_Channel.cpp`

Reply:
the implementation should (for now) be the same in copter and plane, so why not have it in common code?

Reply:
@IamPete1 Or did you mean in the RC_Channel library file, and then both vehicles can override it in their own RC_Channel.cpp files if needed later, but currently they wouldn't, so it'd be the same for both?

Reply:
duplicating the code when it is the same for each doesn't seem like a win